django_multitenant/backends/postgresql/base.py

```diff
@@ -22,15 +16,10 @@ class DatabaseSchemaEditor(PostgresqlDatabaseSchemaEditor):
     sql_create_column_inline_fk = None
     # Override
     def __enter__(self):
-        ret = super(DatabaseSchemaEditor, self).__enter__()
+        ret = super().__enter__()
         return ret
 
-    # Override
-    """
-    If there is a change in the field, this method assures that if the field is type of TenantForeignKey
-    and db_constraint does not exist, adds the foreign key constraint.
-    """
-
+    # pylint: disable=too-many-arguments
```

Comment:
Since this method is Django's own methods, we can not change the signature so I disabled it